noir/noir-repo/compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_slice_ops.rs

```diff
@@ -13,38 +13,50 @@ impl<'block> BrilligBlock<'block> {
         variables_to_insert: &[BrilligVariable],
```

Comment:
I'll extract slice ops into procedures after this PR, they have gotten big enough at this point. It's tricky because you can have slice ops that operate in a different amount of variables (to insert a composite element) but I have a plan for that too